### PR TITLE
MC-11640: doc for retrieve and delete subnet

### PR DESCRIPTION
**Pull request by AnKhangC**
#### Changes made

- ...

#### Related PRs
- []()

### CODE REVIEW

**Review by lamminade — APPROVED** [review]

File: source/includes/gcp/_subnets.md [hunk 1]
```diff
@@ -97,4 +148,12 @@ Required | &nbsp;
...
\ No newline at end of file
+`description`<br/>*string* | Description of the subnet
+
+<!-------------------- DELETE A SUBNET -------------------->
+
+#### Delete a subnet
+
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/subnetworks/:id</code>
+
+Delete an existing subnet
```
Comment (fyrnaga) [inline source/includes/gcp/_subnets.md:165]: Missing an example call as the others

File: source/includes/gcp/_subnets.md [hunk 2]
```diff
@@ -62,7 +62,58 @@ Attributes | &nbsp;
...
+`ipCidrRange`<br/>*string* | Primary IP address range for the following resources: VM instances, internal load balancers, and internal protocol forwarding
+`kind`<br/>*string* | Type of the resource
+`network`<br/>*string* | Server-defined URL for the VPC network that contains the subnet
+`privateIpGoogleAccess`<br/>*string* | Whether the Private Google Access is configured
+`region`<br/>*Array[Disk]* | Server-defined URL for the region name
+`selfLink`<br/>*string* | Server-defined URL for this resource
+`shortNetwork`<br/>*string* | Display name of the VPC network that contains the subnet
+`id`<br/>*UUID* | The id of the subnet
+`name`<br/>*string* | The display name of the subnet.
+`shortRegion`<br/>*string* | A short version of the region name
```
Comment (fyrnaga) [inline source/includes/gcp/_subnets.md]: They should all end with a period.

**Review by fyrnaga — COMMENTED** [review]
Small corrections but  no blocker

File: source/includes/gcp/_subnets.md (hunk 2 above)
Comment (AnKhangC) [reply source/includes/gcp/_subnets.md]: afa6497

File: source/includes/gcp/_subnets.md (hunk 1 above)
Comment (AnKhangC) [reply source/includes/gcp/_subnets.md:165]: afa6497

**Review by fyrnaga — APPROVED** [review]
LGTM

**Pull request merged**